tests/p2p/test_tx_pool.py

```diff
@@ -93,7 +131,35 @@ def finalizer():
 
 
 @pytest.mark.asyncio
-async def test_tx_sending(request, event_loop):
+async def test_does_not_propagate_invalid_tx(monkeypatch,
+                                             request,
+                                             event_loop,
+                                             chain_with_block_validation,
+                                             tx_validator):
+
+    peer1, peer1_txs_recorder, peer2, peer2_txs_recorder, pool = await bootstrap_test_setup(
+        monkeypatch,
+        request,
+        event_loop,
+        chain_with_block_validation,
+        tx_validator
+    )
+
+    txs_broadcasted_by_peer1 = [
+        create_random_tx(chain_with_block_validation, False),
```

Comment:
I'm allergic to boolean literals as positional arguments.  Can you update to calling this using a keyword so it is clear what the `False` does here?

Reply:
Agree. Need to get used to keyword arguments. :+1: